src/directory/generateDirectory.mjs

```diff
@@ -64,11 +66,22 @@ async function traverseDirectoryObject(directoryNode) {
           );
         }
 
+        // Relative file path from the `src/pages` directory
+        // This helps us create the paths like "/[platform]/..." and "/gen2/..."
         const relativePath = path.relative(rootPath, directoryNode.path);
-        const parsedPath = path.parse(relativePath);
+
+        // Convert the relative path, that could be POSIX or Windows,
+        // into a POSIX path that contains forward slashes for the path separator.
+        // This is so we can let the default paths correctly find the file paths and then
+        // use the forward slashes to help set up the route for Next.js
+        const posixRelativePath = path.posix.join(
+          ...relativePath.split(path.sep)
+        );
```

Comment:
Created an amazon workspaces to get access to a Windows machine and was able to test this out and get everything running in my windows local environment!~